### ISSUE TITLE
Add source code linting

### ISSUE THREAD

**Issue by peterlazar1993** [m-arav/Kitsunae#3]
Best to use eslint. Let's discuss on what rules to enforce.

**Comment by m-arav** [comment]
@peterlazar1993 Yes ,we should discuss on the rules for eslint before adding it in
